src/agent/Server.cpp

```diff
@@ -71,12 +72,25 @@ void Server::setOverallStatus(GeneralHealth::Health overallStatus)
     emit overallStatusChanged(m_health);
 }
 
+void Server::setDiskInfoCollection(std::vector<DiskInfo> diskInfoCollection)
```

Comment:
const &

Reply:
This is an override method from rep_AgentStatus_source.h which is generated automatically.